tables/tables_schemas.md

```diff
@@ -38,10 +38,10 @@ interface TableColumn {
 ```typescript
 enum TableColumnDataType {
   Number = "Number",
-  FixedLengthText = "Fixed length text",
-  TrueFalse = "True/False",
+  FixedLengthText = "FixedLengthText",
```

Comment:
@myfoxtail I think it should be `"fixedLengthText"`. We are requesting with small letter as a first letter. Response with capital letter. Same for other types too.